### ISSUE TITLE
[android][sdk 47][expo-av 13.0.3] Compile failure

### ISSUE THREAD

**Issue by zhiqingchen** [expo/expo#21480]
### Minimal reproducible example

https://github.com/zhiqingchen/taro-test/actions/runs/4310086585/jobs/7518170953

### Summary

When I upgrade the expo-av version from 13.0.2 to 13.0.3 the compilation fails

Here is the error message
```
1: Task failed with an exception.
-----------

* Where:
Build file '/home/runner/work/taro-test/taro-test/node_modules/expo-av/android/build.gradle' line: 181

* What went wrong:
A problem occurred evaluating project ':expo-av'.
> Could not find method applyLegacyReactNativeLibsExtractionPlugin() for arguments [] on project ':expo-av' of type org.gradle.api.Project.

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
```

When I locked the version to 13.0.2, it compiled fine

https://github.com/zhiqingchen/taro-test/actions/runs/4310133127/jobs/7518268425

### Environment
```
  expo-env-info 1.0.5 environment info:
    System:
      OS: macOS 13.0
      Shell: 5.8.1 - /bin/zsh
    Binaries:
      Node: 18.13.0 - /usr/local/bin/node
      Yarn: 1.22.19 - ~/.yarn/bin/yarn
      npm: 8.19.3 - ~/.nvm/versions/node/v18.13.0/bin/npm
      Watchman: 2023.02.13.00 - /opt/homebrew/bin/watchman
    Managers:
      CocoaPods: 1.11.3 - /usr/local/bin/pod
    SDKs:
      iOS SDK:
        Platforms: DriverKit 22.1, iOS 16.1, macOS 13.0, tvOS 16.1, watchOS 9.1
      Android SDK:
        API Levels: 28, 29, 30, 31, 33
        Build Tools: 28.0.3, 29.0.2, 30.0.2, 30.0.3, 31.0.0, 33.0.0
        System Images: android-30 | Google APIs Intel x86 Atom
    IDEs:
      Android Studio: 2021.2 AI-212.5712.43.2112.8512546
      Xcode: 14.1/14B47b - /usr/bin/xcodebuild
    npmPackages:
      expo: ~47.0.3 => 47.0.13
      react: ^18.1.0 => 18.2.0
      react-dom: ^18.1.0 => 18.2.0
      react-native: ^0.70.1 => 0.70.7
    Expo Workflow: bare
```

**Comment by Kudo** [comment]
hi there! please try to remove the [`expo-modules-core`](https://github.com/zhiqingchen/taro-test/blob/3c7d326f6d38883c9178721097d39369efafada0/package.json#L79) which should only be a transitive dependency of the `expo` package.

**Comment by zhiqingchen** [comment]
thanks, [it works ](https://github.com/zhiqingchen/taro-test/actions/runs/4322377897/jobs/7544720835)

**Comment by ShumGG** [comment]
@Kudo i also updated expo av to 13.0.3 but i need to use expo-modules-core because im using ```DeviceEventEmitter```, is there another solution?

**Comment by ShumGG** [comment]
> @Kudo i also updated expo av to 13.0.3 but i need to use expo-modules-core because im using `DeviceEventEmitter`, is there another solution?

solved, updated expo modules core to 1.1.1 with expo-av 13.0.3 for sdk 47
